internal/plugins/openshift/v1/init.go

```diff
@@ -27,7 +27,7 @@ import (
 
 const (
 	// The current OCP release version.
-	ocpProductVersion = "4.15"
+	ocpProductVersion = "4.18"
 	// The currently used version of ubi8/ubi-minimal images.
 	ubiMinimalVersion = "8.8"
```

Comment:
Is there any update required for ubi version?

Reply:
The image version has known security vulnerabilites.

Reply:
The latest ubi8 tag available in `registry.access.redhat.com/ubi8/ubi-minimal` and `registry.access.redhat.com/ubi8/ubi-micro` is 8.10. Should I update to 8.10?

Reply:
Updated to 8.10.